Dockerfile

```diff
@@ -21,4 +21,30 @@ COPY --from=builder /workspace/vendor/ /vendor/
 RUN chgrp -R 0 /vendor && chmod -R g=u /vendor
 RUN chgrp -R 0 /environments && chmod -R g=u /environments
 RUN chgrp -R 0 /components && chmod -R g=u /components
+
+ARG BUILD_DATE
+ARG VERSION
+ARG VCS_REF
+ARG DOCKERFILE_PATH
+ARG VCS_BRANCH
+
+LABEL vendor="Observatorium" \
+    name="observatorium/configuration" \
+    description="Observatorium Operator" \
+    io.k8s.display-name="observatorium/configuration" \
```

Comment:
all of these should rather be `observatorium/observatoriumoperator` no?

Reply:
Done

Reply:
Done